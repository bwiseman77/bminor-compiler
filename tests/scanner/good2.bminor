"this is a \fancy \'string\' :)"
" wow this is so\nmuch\nFUN right mia"
integer
a: fing;
'q';
'\a';
'\0'
'\n'
