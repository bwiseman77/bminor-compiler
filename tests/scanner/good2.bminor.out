-------------------------------------------
           token type  text
-------------------------------------------
      STRING_LITERAL  this is a fancy 'string' :)
      STRING_LITERAL   wow this is so
much
FUN right mia
             INTEGER  integer
               IDENT  a
               COLON  :
               IDENT  fing
           SEMICOLON  ;
        CHAR_LITERAL  q
           SEMICOLON  ;
        CHAR_LITERAL  a
           SEMICOLON  ;
        CHAR_LITERAL  
        CHAR_LITERAL  

